pkgs/development/python-modules/graph-tool/2.x.x.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, python, cairomm, sparsehash, pycairo, autoreconfHook
-, pkgconfig, boost, expat, scipy, cgal, gmp, mpfr
+, pkg-config, boost, expat, scipy, cgal, gmp, mpfr
```

Comment:
You also need to fix the parameter in pkgs/top-level/python-packages.nix to fix the eval